style/outsidein.css

```diff
@@ -442,4 +455,106 @@ body {
+    background-color: #fff !important;
+    color: #000 !important;
+    outline: none;
+}
+
+.fl-theme-wb .outside-in-container .outside-in-listRowStyle1:before,
+.fl-theme-wb .outside-in-container .outside-in-listRowStyle2:before {
+    background-color: #fff;
+}
+
+
+.fl-theme-yb .outside-in-container .outside-in-headerLinks a:hover,
+.fl-theme-yb .outside-in-container .outside-in-partnersRow a:hover,
+.fl-theme-yb .outside-in-container .outside-in-emailLink:hover,
+.fl-theme-yb .outside-in-container .outside-in-footerLogo a:hover:before,
+.fl-theme-yb .outside-in-container .outside-in-footerLogo a:hover,
+.fl-theme-yb .outside-in-container .outside-in-iconColumn:before,
+.fl-theme-yb .outside-in-container a:focus,
+.fl-theme-yb .outside-in-container a:focus:before,
+.fl-theme-yb .fl-prefsEditor-buttons button:focus,
+.fl-theme-yb .fl-prefsEditor-buttons button:hover {
+    background-color: #ffff00 !important;
+    color: #000 !important;
+    outline: none;
+}
+
+.fl-theme-yb .outside-in-container .outside-in-listRowStyle1:before,
+.fl-theme-yb .outside-in-container .outside-in-listRowStyle2:before {
+    background-color: #ffff00;
+}
+
+
+.fl-theme-by .outside-in-container .outside-in-headerLinks a:hover,
+.fl-theme-by .outside-in-container .outside-in-partnersRow a:hover,
+.fl-theme-by .outside-in-container .outside-in-emailLink:hover,
+.fl-theme-by .outside-in-container .outside-in-footerLogo a:hover:before,
+.fl-theme-by .outside-in-container .outside-in-footerLogo a:hover,
+.fl-theme-by .outside-in-container .outside-in-iconColumn:before,
+.fl-theme-by .outside-in-container a:focus,
+.fl-theme-by .outside-in-container a:focus:before,
+.fl-theme-by .fl-prefsEditor-buttons button:focus,
+.fl-theme-by .fl-prefsEditor-buttons button:hover {
+    background-color: #000 !important;
+    color: #ffff00 !important;
+    outline: none;
+}
+
+.fl-theme-by .outside-in-container .outside-in-listRowStyle1:before,
+.fl-theme-by .outside-in-container .outside-in-listRowStyle2:before {
+    background-color: #000;
+}
+
+.fl-theme-lgdg .outside-in-container .outside-in-headerLinks a:hover,
+.fl-theme-lgdg .outside-in-container .outside-in-partnersRow a:hover,
+.fl-theme-lgdg .outside-in-container .outside-in-emailLink:hover,
+.fl-theme-lgdg .outside-in-container .outside-in-footerLogo a:hover:before,
+.fl-theme-lgdg .outside-in-container .outside-in-footerLogo a:hover,
+.fl-theme-lgdg .outside-in-container .outside-in-iconColumn:before,
+.fl-theme-lgdg .outside-in-container a:focus,
+.fl-theme-lgdg .outside-in-container a:focus:before,
+.fl-theme-lgdg .fl-prefsEditor-buttons button:focus,
+.fl-theme-lgdg .fl-prefsEditor-buttons button:hover {
+    background-color: #bdbdbb !important;
+    color: #555 !important;
+    outline: none;
+}
+
+.fl-theme-lgdg .outside-in-container .outside-in-listRowStyle1:before,
+.fl-theme-lgdg .outside-in-container .outside-in-listRowStyle2:before {
+    background-color: #bdbdbb;
```

Comment:
We need a style like this for the similar "before" rectangles on H3s (and any other header level)